samples/matter/closure/README.rst

```diff
@@ -0,0 +1,309 @@
+.. _matter_closure_sample:
+
+Matter: Closure
+###############
+
+.. contents::
+   :local:
+   :depth: 2
+
+This sample demonstrates the usage of the :ref:`Matter <ug_matter>` application layer to build a closure device.
+This device works as a Matter accessory device, meaning it can be paired and controlled remotely over a Matter network built on top of a low-power 802.15.4 Thread network or on top of a Wi-Fi® network.
+Additionally, this device works as a :ref:`Full Thread Device (FTD) <thread_ot_device_types>`.
+Support for both Thread and Wi-Fi is mutually exclusive and depends on the hardware platform, so only one protocol can be supported for a specific closure device.
+You can use this sample as a reference for creating your own application.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+If you want to commission the closure device and :ref:`control it remotely <matter_window_cover_network_mode>` through a Thread network, you need to set-up the :ref:`Thread Border Router <ug_thread_tools_tbr>` and control it with the :ref:`CHIP Tool <chip_tool_guide>`.
+Later support for this device might be added to ecosystems.
+When this happens you will also be able to control it through a Matter controller device :ref:`configured on PC or smartphone <ug_matter_configuring>`.
+This requires additional hardware depending on the setup you choose.
+
+.. note::
+    |matter_gn_required_note|
+
+IPv6 network support
+====================
+
+The development kits for this sample offer the following IPv6 network support for Matter:
+
+* Matter over Thread is supported for the ``nrf52840dk/nrf52840``, ``nrf5340dk/nrf5340/cpuapp``, ``nrf54l15dk/nrf54l15/cpuapp`` and ``nrf54lm20dk/nrf54lm20a/cpuapp`` board targets.
+* Matter over Wi-Fi is supported for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target with the ``nrf7002eb2`` shield attached.
+
+Overview
+********
+
+The sample starts the Bluetooth® LE advertising automatically and prepares the Matter device for commissioning into a Matter-enabled Thread network.
```

Comment:
I think we don't need to put this here for two reasons:
1. It's not that important to put it as the first sentence of overview
2. It's repeated then in sections related to commissioning